src/Features/CSharp/Portable/CodeFixes/PopulateSwitch/PopulateSwitchCodeFixProvider.PopulateSwitchFixAllProvider.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeFixes.PopulateSwitch
+{
+    internal partial class PopulateSwitchCodeFixProvider : CodeFixProvider
+    {
+        private class PopulateSwitchFixAllProvider : BatchSimplificationFixAllProvider
+        {
+            internal static new readonly PopulateSwitchFixAllProvider Instance = new PopulateSwitchFixAllProvider();
```

Comment:
I derived from `BatchSimplificationFixAllProvider` because the IDE analyzers did.